.github/workflows/ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
     - name: Run chart-testing (lint)
       id: lint
-      uses: helm/chart-testing-action@v1.0.0
+      uses: helm/chart-testing-action@b0d4458c71155b54fcf33e11dd465dc923550009
```

Comment:
The action fails

```
Requirement already satisfied: pyyaml in /opt/hostedtoolcache/ct/v3.3.0/x86_64/venv/lib/python3.6/site-packages (from yamale==3.0.4)
Building wheels for collected packages: yamale
  Running setup.py bdist_wheel for yamale: started
  Running setup.py bdist_wheel for yamale: finished with status 'error'
  Complete output from command /opt/hostedtoolcache/ct/v3.3.0/x86_64/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-c2y5j063/yamale/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpgjeo_gsgpip-wheel- --python-tag cp36:
  /usr/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
    warnings.warn(msg)
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help
  
  error: invalid command 'bdist_wheel'
  
  ----------------------------------------
  Running setup.py clean for yamale
  Failed building wheel for yamale
```

Reply:
Fixed it by updating python version to 3.7😉

Reply:
Reference: https://github.com/helm/chart-testing-action/issues/65